### PR TITLE
Document and add `update_display` to Host

### PR DESCRIPTION
**Pull request by frabert**
Reopens #125

I'm not currently using vst-rs anymore, so I won't be able to properly test this. I'll gladly accept fixes and suggestions before merging.

cc @askeksa

### CODE REVIEW

File: src/host.rs [hunk 1]
```diff
@@ -222,6 +224,9 @@ pub trait Host {
     fn get_block_size(&self) -> isize {
         0
     }
+
+    /// Refresh UI after the plugin's parameters changed.
```
Comment (askeksa) [inline src/host.rs:228]: Add to doc comment:

Note: some hosts will call some `PluginParameters` methods from within the `update_display` call, including `get_parameter`, `get_parameter_label`, `get_parameter_name` and `get_parameter_text`.

**Comment by frabert** [comment]
I made the changes to the documentation, but it seems that some problems may be related to recent changes in Rust's API or style guides.

**Comment by askeksa** [comment]
Clippy issues fixed in https://github.com/RustAudio/vst-rs/commit/d608f3e649df0ab4d417f01eb4a7f80b73c37045. Try rebasing your branch onto this commit. That should make it pass the CI.

**Comment by frabert** [comment]
Done!

**Pull request merged**